src/platform_impl/linux/window.rs

```diff
@@ -314,6 +314,86 @@ impl Window {
     Ok(win)
   }
 
+  /// Create a new Tao window from an existing GTK window. Generally you should use
+  /// the non-Linux `WindowBuilder`, this is for those who need lower level window access
+  /// and know what they're doing.
+  pub fn new_from_gtk_window<T>(
```

Comment:
marking this function public here doesn't mean it is actually public, you will need to expose it on `WindowExtUnix` trait in https://github.com/tauri-apps/tao/blob/dev/src/platform/unix.rs

Reply:
Done.